### ISSUE TITLE
False error on CRLdp does not contain url

### ISSUE THREAD

**Issue by rmhrisk** [zmap/zlint#223]
I believe there may be a bug in Zlint. If you look at this certificate:
https://censys.io/certificates/6ef0debdfc2d9af68c5174c2b88ccfabde9540a1b6b96e2526b59e6a9ab09b10

You will see it has a zlint error, specifically, it is:
```
sub cert crl distribution points does not contain url
```

And if you look at the CRLdp in the CenSys user interface it shows a single ldap URI:
![image](https://user-images.githubusercontent.com/1619279/40448587-3a6dd324-5ed6-11e8-8774-24d45233ac7d.png)

But if you look at the same certificate under OpenSSL you see:
![image](https://user-images.githubusercontent.com/1619279/40448611-4ef2101c-5ed6-11e8-8570-e03511aef166.png)

Upon inspection of the lint code, it seems it is doing the right things:
https://github.com/zmap/zlint/blob/master/lints/lint_sub_cert_crl_distribution_points_does_not_contain_url.go#L43

So I guess the X.509 decoder is only returning a single CRLdp?

**Comment by justinbastress** [comment]
From https://www.ietf.org/rfc/rfc5280.txt:

```
   If the DistributionPointName contains multiple values, each name
   describes a different mechanism to obtain the same CRL.  For example,
   the same CRL could be available for retrieval through both LDAP and
   HTTP.
```

So unless the CABR says otherwise, it seems like this error is indeed erroneous.

**Comment by rmhrisk** [comment]
@justinbastress Thanks for the quick response!

The CABR has nothing on this, beyond a HTTP link must be present.

**Comment by nguyekim** [comment]
It seems to me that the RFC - as in many cases - leaves room for interpretation and that both options seem valid, i.e. the formulation as quoted above from the RFC allows both.

**Comment by rmhrisk** [comment]
In such cases what clients support is a good litmus test. Windows, OSX, OpenSSL, and probably others seem to like the nested approach.

**Comment by nguyekim** [comment]
Just for asking for clarification, the nested Approach refers to which coding option mentioned above? 
CRLDistributionPoints: [ { fullname: [ "url1", "url2", "url3" ] } ] or the secondOption. E.g. Windows displays this first option correctly.

**Comment by rmhrisk** [comment]
Yes, if I understand your question correctly I mean the encoding.

**Comment by nguyekim** [comment]
Thanks for quickly solving this issue! Kim

**Comment by sdavidson-quovadis** [comment]
Thank you for resolving this!
